packages/@lightningjs/ui-components/src/components/Icon/Icon.js

```diff
@@ -165,8 +165,6 @@ export default class Icon extends Base {
     return {
       texture,
       shader,
-      w: this.w,
-      h: this.h,
```

Comment:
why were these removed?

Reply:
If these are left in place there is an infinite loop in badge icon with pngs

Reply:
Coooool